client/client-multi/client-cpp/test/impl/ClientStatusTest.cpp

```diff
@@ -57,6 +75,8 @@ BOOST_AUTO_TEST_CASE(checkDefaults)
 
 BOOST_AUTO_TEST_CASE(checkSetAndSaveParameters)
 {
+//    auto stateMock = std::make_shared<MockKaaClientStateStorage>();
+//    KaaClientContext clientContext(properties, tmp_logger, *stateMock, context);
```

Comment:
Do not leave commented code for no reason. You always can use the SCM to restore any piece of code that you want.